compiler/rustc_index/src/bit_set.rs

```diff
@@ -752,6 +733,7 @@ impl<T: Idx> BitRelations<ChunkedBitSet<T>> for ChunkedBitSet<T> {
         changed
     }
 
+    // njn: add test for this
```

Comment:
Maybe worth doing in this PR?

Reply:
Those two functions were implemented in #131326 and it would be good to have tests, but no need to hold this PR up on that.